src/connection/activate.ts

```diff
@@ -42,6 +42,15 @@ function useTryActivation() {
 
         console.debug(`Connection activating: ${connection.getName()}`)
         dispatch(updateSelectedWallet({ wallet: undefined }))
+        console.log('connection', connection)
+        // console.log('activeConnector', connection.connector.deactivate)
```

Comment:
uncomment? 👀

Reply:
accidentally included my debugging of WC, resolved